common/presenters/table/object-to-table-head.js

```diff
@@ -1,17 +1,61 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
-function objectToTableHead({ head } = {}) {
-  if (!head) {
-    return undefined
+function buildUrl({ key }, query = {}) {
+  return queryString.stringify({
+    ...query,
+    sortBy: key,
+    sortDirection: query.sortDirection === 'asc' ? 'desc' : 'asc',
+  })
+}
+
+function buildAriaLabel(head, query) {
+  // the label indicates the result of the sorting action triggered by the link,
+  // which is the opposite of the current state
+  return `${i18n.t('sort')} ${i18n.t(head.html).toLowerCase()} ${
```

Comment:
It might be neater to have this as one translation key where you pass in the label and use `context` for the direction:

```json
// translation
"sort_ascending": "Sort {{label}} ascending",
"sort_descending": "Sort {{label}} descending"
```

```javascript
return i18n.t('sort', {
  context: query.sortDirection === 'asc' ? i18n.t('descending') : i18n.t('ascending'),
  label: i18n.t(head.html),
})
```

Reply:
👍